### DISCUSSION TITLE
can't show model summary in kaggle's notebook

### DISCUSSION THREAD

**Discussion by lg2578** [Lightning-AI/pytorch-lightning#13258] [General]
my code :
trainer = pl.Trainer(
    max_epochs=155,
    accelerator='auto',
    gpus=1 if torch.cuda.is_available() else None, 
    callbacks=[save_best_cb,swa_cb],
)
model = pl_classifier(net)
trainer.fit(model, train_dataloaders=train_dl,val_dataloaders=val_dl)

it show a model summary under vscode's jupyter notebook and colab's,
but show nothing in kaggle's

**Comment by akihironitta** [comment]
Hi @lg2578 thank you for reporting this! It sounds like an unexpected behaviour. Would you mind creating an issue with a reproducible script and env information? https://github.com/Lightning-AI/lightning/issues/new/choose
